README.md

```diff
@@ -16,9 +18,47 @@ aims to apply the same technique for bootable host systems - using
 standard OCI/Docker containers as a transport and delivery format
 for base operating system updates.
 
-## More information
+## Generated container images
+
+At the moment the main image is `quay.io/centos-bootc/centos-bootc:stream9`
+
+## Building
```

Comment:
OK, I hadn't paid attention to this PR until now but this section is similar to what I propose on the Fedora side in https://gitlab.com/fedora/bootc/base-images/-/merge_requests/6. I think (as done there) it'd be good to link to the relevant docs pages and mention that layering is the expected flow. WDYT about taking what I have there here, and moving the Mac bit into the docs so that the README stays shorter?